dash/frontend/src/app/modules/private/pages/reports/running-vulnerabilities/running-vulnerabilities.component.html

```diff
@@ -70,14 +70,16 @@
       <mat-card appearance="outlined">
```

Comment:
Remove `appearance="outlined"` if you don't mind :)

Reply:
Also if the table should be scrollable, we have classes for that.

Reply:
If it's scrolling properly on a small screen, that's good enough for me.

Reply:
You should double check to be sure.
Here I have a <mat-card-content> with a nested <mat-table>:
![image](https://github.com/m9sweeper/m9sweeper/assets/50845926/0390f9eb-2735-44ab-9c70-eb8548ca3dac)

Reply:
When it's nested like:
`        <mat-card class="scrollable-table-card">`
`          <mat-card-content>`
`            <div class="table-wrapper">`
`              <mat-table [dataSource]="dataSource" id="falco-table">`
and it has scss setting the min-widths of each column,
it works

Reply:
The tables scroll fine on small screens locally.

![image](https://github.com/m9sweeper/m9sweeper/assets/47117895/f2b72322-1595-4f00-afa5-0d4ec1d2b2b9)

If I remember right the issue was the card itself had a min width set at 800px, removing that and it sizes down as expected and mat-card-content handles the table scrolling.